.github/workflows/upgrade-dependencies.yaml

```diff
@@ -0,0 +1,13 @@
+name: Create Upgrade-Pull-Requests
```

Comment:
Similar to https://github.com/gardener/gardener/pull/12592#discussion_r2224593170, we could drop this workflow. This repo is using dependabot.